configure

```diff
@@ -8073,6 +8079,20 @@ then :
 fi
 
 
+# Check whether --enable-prgname-prefix was given.
```

Comment:
In line 1539 you defined `--disable-prgname-prefix`, but here you are checking `--enable-prgname-prefix`. Is this the correct logic?